lib/components/viewers/trip-viewer.js

```diff
@@ -98,11 +112,18 @@ class TripViewer extends Component {
               else if (i > viewedTrip.fromIndex && i < viewedTrip.toIndex) highlightClass = 'strip-map-highlight'
               else if (i === viewedTrip.toIndex) highlightClass = 'strip-map-highlight-last'
 
+              // Convert to unix (millisceonds) for FormattedTime
+              var now = new Date()
+              var startOfDay = new Date(now.getFullYear(), now.getMonth(), now.getDate())
```

Comment:
See remark about this repeated code block.

Reply:
Changed to `const` in new methodology. The use in trip-viewer is slightly different but lmk and I can refactor into a function.